### ISSUE TITLE
API documentation broken

### ISSUE THREAD

**Issue by twagoo** [clarin-eric/component-registry-rest#20]
See [ComponentRegistry/api-docs](https://catalog.clarin.eu/ds/ComponentRegistry/api-docs):

```
Can't read swagger JSON from https://catalog.clarin.eu/ds/ComponentRegistry/api-docs/../../rest/api-docs
```

Probably the URI is misconfigured. [ComponentRegistry/rest/api-docs](https://catalog.clarin.eu/ds/ComponentRegistry/rest/api-docs/):

``` json
{
"apiVersion":"1.0.0",
"swaggerVersion":"1.2",
"apis":[{"path":"/authentication","description":"REST resource for handling the authentication status"}]
}
```

**Comment by twagoo** [comment]
JSON location fixed in fc84be131834c0b45ef038ee7c6830fc80980f59
Registry REST resource still not showing up, probably due to https://github.com/swagger-api/swagger-core/issues/102

**Comment by twagoo** [comment]
Fixed as of 00826c749f13a89bb997519f32087076b188b7bd after a swagger version upgrade (servlet and front end) and additional annotations on 'auxiliary' REST resources (`/items` and `/groups`)
